sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/output.py

```diff
@@ -153,3 +155,46 @@ def _assert_name_and_version(self):
             )
         if self.version and not self.name:
             raise UserErrorException("Output name is required when output version is specified.")
+
+
+class OutputMetadata(object):
```

Comment:
Why is it named OutputMetadata? Isn't is supposed to be used for both input and output?

Reply:
I am also curious on this question

Reply:
done